data/json/items/containers.json

```diff
@@ -256,11 +256,11 @@
     "looks_like": "bottle_plastic",
     "description": "A resealable glass bottle, holds 750 ml of liquid.",
     "weight": "450 g",
-    "volume": "1734 ml",
+    "volume": "1318 ml",
     "longest_side": "276 mm",
     "price": 0,
     "price_postapoc": 10,
-    "to_hit": 1,
+    "to_hit": 1,Fglass
```

Comment:
Accidental paste?

Reply:
Failed search (shift+f instead of ctrl+f). Thanks for catching it.